lumigator/python/mzai/jobs/evaluator/evaluator/jobs/evaluation/hf_evaluate.py

```diff
@@ -123,8 +123,12 @@ def run_eval(config: HuggingFaceEvalJobConfig) -> Path:
             logger.info(f"Using direct HF model invocation. Model: {model_name}")
             model_client = HuggingFaceModelClient(model_name, config)
 
-    # run inference
-    predictions, summarization_time = predict(dataset_iterable, model_client)
+    if not config.evaluation.skip_inference:
```

Comment:
Nit, as discussed... 
if skip: take from dataset
else: predict(...) 
?

Reply:
TY, fixed in [here](https://github.com/mozilla-ai/lumigator/pull/540/commits/6ea76eed2049e471f2dc36d0a83b60f75bbbbf2e)